### PR TITLE
Add OpenShift Route resources to product documentation.

### PR DESCRIPTION
**Pull request by f5gary**
Problem:
The product documention needs to be updated to cover OpenShift Route
resources.

Solution:
Added key information regarding OpenShift Routes to the product
documentation, including examples. Removed hidden attribute from new
controller parameters.

affects-branches: master

### CODE REVIEW

File: docs/README.rst [hunk 1]
```diff
@@ -301,7 +321,7 @@ Backend
 
 Ingress Resources
 -----------------
-The |kctlr-long| supports Kubernetes Ingress resources as an alternative to F5 Resource ConfigMaps.
+The |kctlr-long| supports Kubernetes Ingress resources as an alternative to F5 Resource ConfigMaps and Kubernetes Ingress Resources.
```
Comment (sjberman) [inline docs/README.rst]: Did you mean "and OpenShift Route Resources" here?

File: docs/README.rst [hunk 2]
```diff
@@ -344,6 +364,40 @@ To configure health monitors on your Ingress resource, you need to use the appro
...
+
++-------------------------+-------------------+-------------------+---------+-----------------+-------------------------------------------------------------------------+
+| Type                    | Client Connection | Server Connection | Path    | SSL Termination | Notes                                                                   |
+|                         | Encrypted         | Encrypted         | Support | on BIG-IP       |                                                                         |
++=========================+===================+===================+=========+===========================================================================================+
+| Unsecured               | No                | No                | Yes     | No              | The BIG-IP system forwards unsecured traffic from the client to the     |
+|                         |                   |                   |         |                 | endpoint.                                                               |
++-------------------------+-------------------+-------------------+---------+-----------------+-------------------------------------------------------------------------+
```
Comment (sjberman) [inline docs/README.rst]: When viewing this file there seems to be some formatting issues in this first section of the table, though looking at the code I can't exactly tell what is causing it.

File: docs/README.rst (hunk 1 above)
Comment (f5gary) [reply docs/README.rst]: Yes, will fix.

File: docs/README.rst (hunk 2 above)
Comment (f5gary) [reply docs/README.rst]: I found it, will fix.

**Pull request merged**